content/integrations/awss3.md

```diff
@@ -9,15 +9,25 @@ git_integration_title: amazon_s3
 
 # Overview
 
+![RDS Dashboard](/static/images/s3_db_screenshot.png)
+
 Amazon Simple Storage Service (S3) is a highly available and scalable cloud storage service.
 
 Enable this integration to see in Datadog all your S3 metrics.
 
 # Installation
 
-The only requirement for this integration is the permission `s3:GetBucketTagging`.
+* **Daily Storage Metrics**
+
+	The only requirement for this integration is the permission `s3:GetBucketTagging`.
+
+* **Request Metrics**
```

Comment:
I guess I should reword that. If you only want daily S3 metrics, you only have to have that permission in the AWS policy